posthog/hogql/database/schema/sessions_v1.py

```diff
@@ -230,7 +236,10 @@ def arg_max_merge_field(field_name: str) -> ast.Call:
                             # if session duration >= 10 seconds, not a bounce
```

Comment:
Let's update the comment

Btw, we were using 10 seconds, not 30, lol, website docs are probably outdated

Reply:
https://github.com/PostHog/posthog.com/pull/10206